dotnet-authserver/src/TeacherIdentity.AuthServer/AuthenticationState.cs

```diff
@@ -84,6 +84,7 @@ public class AuthenticationState
     public string? TrnOwnerEmailAddress { get; private set; }
     [JsonInclude]
     public TrnLookupStatus? TrnLookupStatus { get; private set; }
+    public bool HaveNationalInsuranceNumber { get; set; }
```

Comment:
I see everything else has a private setter, is there a reason we shouldn't make this public?

Reply:
If we're setting directly rather than via a `SetXxx` method then that's fine.